exhibitor-core/src/main/resources/com/netflix/exhibitor/core/ui/js/exhibitor-explorer.js

```diff
@@ -119,11 +120,13 @@ function initExplorer()
                     success:function (data){
                         explorerSelectedPath = node.data.key;
                         explorerSelectedBytes = data.bytes;
+                        explorerSelectedACLs = data.aclsArray;
 
                         $("#path").text(node.data.key);
                         $("#stat").text(data.stat);
                         $("#data-bytes").text(data.bytes);
                         $("#data-str").text(data.str);
+                        $("#node-acl").html(data.acls)
```

Comment:
Assuming we want `data.acl` to be json list of strings.
This part will need to be changed to parse it into html.